manifests/virtualenv.pp

```diff
@@ -169,6 +169,17 @@
       require     => File[$venv_dir],
     }
 
+
+    exec { "python_distribute_${venv_dir}":
+      command     => "${pip_cmd} wheel --help > /dev/null 2>&1 && { ${pip_cmd} wheel --version > /dev/null 2>&1 || wheel_support_flag='--no-use-wheel'; } ; { ${pip_cmd} --log ${venv_dir}/pip.log install ${pypi_index} ${proxy_flag} \$wheel_support_flag --upgrade ${distribute_pkg} || ${pip_cmd} --log ${venv_dir}/pip.log install ${pypi_index} ${proxy_flag}  --upgrade ${distribute_pkg} ;}",
```

Comment:
this can cause a huge amount of trouble. Exec resources in general are tricky and should be avoided. Adding multiple shell commands to one resource is really not the way to go. Can you somehow split this up into multiple resources and add acceptance tests for it?